README.md

```diff
@@ -1,298 +0,0 @@
-# ESP-IDF Components library
```

Comment:
do not delete README.md. instead, you need to modify it.

Reply:
I think It was related with ruby error logs I sent on Windows. After bundle success with WSL, I tried your suggestion about io-console, and try to execute again bundle with CMD. This showed second log. As it was an error, I thought, it didn't change anything... but it was my fault... it cleared it.

I usually use github desktop or vscode git plugin to see commit changes, but certainly, I didn't check README.md content. I saw that README was modified but I supposed that was WSL changes I checked before.

Sorry for that.